### PR TITLE
support for new device

### PR DESCRIPTION
**Pull request by rondoval**
This is adding support for no-name BLE RGB Christmas lights. The differences are:
- different UUIDs
- different turn on command

### CODE REVIEW

File: custom_components/elkbledom/elkbledom.py [hunk 1]
```diff
@@ -165,7 +168,7 @@ def effect(self):
 
     @retry_bluetooth_connection_error
     async def set_white(self, intensity: int):
-        await self._write([0x7e, 0x00, 0x01, intensity, 0x00, 0x00, 0x00, 0x00, 0xef])
+        await self._write([0x7e, 0x00, 0x01, int(intensity*100/255), 0x00, 0x00, 0x00, 0x00, 0xef])
```
Comment (rondoval) [inline custom_components/elkbledom/elkbledom.py:171]: Unsure if this is true for other devices... my one seems to have a range of 0 - 0x64; i.e. 0x64 is max brightness, 0x65 is same as 0x0.

**Comment by dave-code-ruiz** [comment]
Please, can you use led ble integration? (https://www.home-assistant.io/integrations/led_ble/)
I think this integration is best for you, report me if it work for you
Thanks

**Comment by dave-code-ruiz** [comment]
Please, if you have time, test my commit changes in main:

https://github.com/dave-code-ruiz/elkbledom/commit/34813b2e8d244c5410f49276f059b17f085edfd7

I create merge for other strips

Thanks

**Comment by rondoval** [comment]
@dave-code-ruiz 1.0.1 works fine with my strips, thank you.
Regarding led_ble integration - it doesn't support these.

**Pull request closed without merging**